pykickstart/commands/ignoredisk.py

```diff
@@ -151,7 +153,29 @@ class F29_IgnoreDisk(F14_IgnoreDisk):
     removedAttrs = F14_IgnoreDisk.removedAttrs
 
     def parse(self, args):
-        return F8_IgnoreDisk.parse(self, args)
+        retval = FC3_IgnoreDisk.parse(self, args)
+
+        howmany = 0
+        if self.ignoredisk:
+            howmany += 1
+        if self.onlyuse:
+            howmany += 1
+        if self.interactive:
+            howmany += 1
+            warnings.warn(_("Ignoring deprecated option on line %s:  The ignoredisk command "
```

Comment:
I think that this warning should be already produced by the argument parser.

Reply:
I am not able to find the warning anywhere in the logs. Maybe this holds only for deprecated commands?
BTW, the warning will be only displayed in storage module log as we don't forward kickstart parsing warnings from modules to anaconda (just errors). Which is perhaps ok for now.